infra/modules/kubevela-addons/definitions/cardano-wallet-connector.cue

```diff
@@ -0,0 +1,127 @@
+parameter: {
+	network:       *"preview" | "preprod" | "mainnet"
+}
+
+#cardanoWalletConfigs: {
+  volumes: [{
+    name: "wallet-db"
+    persistentVolumeClaim: {
+      claimName: "wallet-db"
+    }
+  }]
+  volumeMounts: [{
+    name:      "wallet-db"
+    mountPath: "/wallet-db"
+    storageClassName: "ebs-sc"
+    resources: {
+      requests: {
+        storage: "10Gi"
+      }
+    }
+  }]
+}
+
+#cardanoNodeConfigs: {
+  volumes: [{
+    name: "ipc"
+    emptyDir: {}
+  }]
+  envs: [{
+    name: "CARDANO_NODE_SOCKET_PATH"
+    value: "/ipc/node.socket"
+  }]
+}
+
+#PatchConfig: {
+  volumes: [#cardanoNodeConfigs.volumes + #cardanoWalletConfigs.volumes][0]
+  envs: [#cardanoNodeConfigs.envs][0]
+}
+
+patch: spec: {
+  // +patchStrategy=retainKeys
+  strategy: {
+    type: "Recreate"
```

Comment:
Does this override or just set a default? If the former, we should change the docs above.

Reply:
overriding, updated above the docs